src/modules/UI/scenes/WalletList/style.js

```diff
@@ -49,7 +49,7 @@ export const styles = {
     fontSize: 44
   },
   currentBalanceBoxBits: {
-    color: THEME.COLORS.WHITE,
+    color: THEME.COLORS.TRANSPARENT,
```

Comment:
why did you change anything on the wallet list?

Reply:
This looks like a mistake. Thank you for the heads up.